### PR TITLE
Citing DGtalTools doc in the DGtal doc mainpage

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

This PR cites the DGtalTools doc in the DGtal doc main page.

# Checklist

- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: src/DGtal/doc/mainpage.dox [hunk 1]
```diff
@@ -40,6 +40,15 @@ many promises to be the common basis for future developments made by
 the digital geometry community.
 
 
+DGtal is associated with some [DGtalTools](http://dgtal.org/tools/)
```
Comment (JacquesOlivierLachaud) [inline src/DGtal/doc/mainpage.dox]: with some => with projects

File: src/DGtal/doc/mainpage.dox [hunk 2]
```diff
@@ -40,6 +40,15 @@ many promises to be the common basis for future developments made by
...
+and [DGtalTools-contrib](http://dgtal.org/tools/). Please have a look
```
Comment (kerautret) [inline src/DGtal/doc/mainpage.dox]: yes they are in the same page

**Comment by kerautret** [comment]
appveyor looks failing due to zlib source download, I change it and PR it

**Comment by kerautret** [comment]
@dcoeurjo I PR on you branch a fix for appveyor (zlib download which fails)

**Comment by dcoeurjo** [comment]
Thanks @kerautret. I'll have a look

**Comment by dcoeurjo** [comment]
Ok ```testPolynomial``` seems to be the test that crashed gcc4.8

**Comment by dcoeurjo** [comment]
I'm merging this one with the removal of the testPolynomial build

**Pull request merged**